### PR TITLE
DTFS2-5412 TFM beta banner and logo

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Changes made: 

* Header logo changed to UKEF logo
* Service name saying UKEF removed
* Added beta banner
* Added primary navigation bar to index.njk file
* Added to e2e tests
* Fixed bug when cloning deal -> facilities -> unissuedToIssuedByMaker from null to blank object to fix crash on submit to UKEF

### CODE REVIEW

File: trade-finance-manager-ui/templates/_macros/header.njk [hunk 1]
```diff
@@ -1,56 +1,27 @@
...
-
-  {% include "_partials/primary-navigation.njk" %}
-
-
-{% endmacro %}
+  <header class="moj-header" role="banner">
+    <div class="moj-header__container">
+      <div class="moj-header__logo">
+        <img
+          src="/assets/images/UK-Export-Finance-logo.svg"
```
Comment (abhi-markan) [inline trade-finance-manager-ui/templates/_macros/header.njk:7]: Missing `alt` and `title` tag from ``.

File: trade-finance-manager-ui/templates/_partials/beta-banner.njk [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+<div class="moj-header__container">
+  {{ govukPhaseBanner({
+    tag: {
+      text: "beta"
+    },
+    attributes: {
+      "data-cy": "beta-banner"
+    },
+    html: 'This is a new service – your <a class="govuk-link" data-cy="beta-feedback-link" href="#">feedback</a> will help us to improve it.'
```
Comment (abhi-markan) [inline trade-finance-manager-ui/templates/_partials/beta-banner.njk:11]: Please add `target='_blank'`. Thanks

File: trade-finance-manager-ui/styles/header.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.ukef-header-logo {
+  height: auto;
+  width: 53px;
```
Comment (abhi-markan) [inline trade-finance-manager-ui/styles/header.scss:3]: How is width calculated, is that based on image size or predefined?

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Have left few comments, thanks.

File: trade-finance-manager-ui/templates/_partials/beta-banner.njk (hunk 2 above)
Comment (Zainzzkk) [reply trade-finance-manager-ui/templates/_partials/beta-banner.njk:11]: done

File: trade-finance-manager-ui/templates/_macros/header.njk (hunk 1 above)
Comment (Zainzzkk) [reply trade-finance-manager-ui/templates/_macros/header.njk:7]: done

**Review by abhi-markan — APPROVED** [review]
Looks good to me, I'll let other review it as well to be on safe side.

**Pull request merged**